### DISCUSSION TITLE
Add more than XYZ three bottom line labels.

### DISCUSSION THREAD

**Discussion by note286** [lvjr/tabularray#353] [Ideas]
Sometimes it is found that only `XYZ` are not enough. So I want to try whether all 26 letters are supported, although only the last few letters are usually used. I found it defines in `\regex_const:Nn \c__tblr_child_name_regex { ^ [X-Z] $ }`. If I change `X-Z` to `A-Z`, it works such as `hline{W}={wd=0.08em}`.

I wonder

- Whether this modification will have other bad effects.
- Can `A-Z` be supported by default in subsequent versions if this modification will not affect the effect of using the current version of the package.
- If the author does not intend to add this directly to subsequent versions. How can I modify `X-Z` to `A-Z` in the `.tex` file without directly modifying the source code of the package?

**Comment by lvjr** [comment]
I am not sure whether it will slow down every `tabularray` document. How about changing it to `U-Z`?

**Reply by note286** [reply]
`U-Z` is also acceptable. Or add a package option to allow modification.

**Reply by lvjr** [reply]
Done.

**Reply by note286** [reply]
Good! Thank you.
